timelapse_saver.py

```diff
@@ -0,0 +1,178 @@
+import requests
+from datetime import datetime, timedelta
+import time
+import json
+import calendar
+import csv
+import sys
+import os
+import argparse
+from requests.sessions import default_headers
+import urllib3
+
+#to disable warnings for not verifying host
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
```

Comment:
Same comments here as the face example:

- single requests session for all api calls,
- refactor mediaBaseURL
- main protection